scripts/config.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# general
+sudo="sudo"
```

Comment:
I couldn't think of a better way to get this functionality in. The idea was - if `hddtemp` needs sudo to run this would be set to sudo, if not it would be left empty. I bet there is a more elegant way to do this...